articles/node-js-vs-django/index.md

```diff
@@ -0,0 +1,83 @@
+Node.js and Django are technologies used in building versatile web and mobile applications. Both have an excellent user interface, are open, and do not require a license to operate, and therefore they have a vast number of users. Node.js is a framework that JavaScript uses on a client-side web application. Django works on [Python applications](https://www.javatpoint.com/python-applications) to help web developers build apps more quickly.
+
+This article will analyze these two popular technologies. The article will critically give each member the pros and cons to help the reader choose the best technology depending on the project needs.
+
+### Node.js
+
+It is an open-source JavaScript run time. The first version was released in the [year 2009](https://nodejs.dev/learn/a-brief-history-of-nodejs). Node.js is based on the following programming languages; C, C++, and JavaScript. Node.js application is on the client-side browser. It provides scalability and concurrency.
```

Comment:
Use the official node.js definition here.

Reply:
@lalith1403 I have included official definition